Before: 48869d1981a00fba39a03f52822e6689dda86fad7eef70686de019a4e257c658
After: 92a69caee8b19d2a677fc4a5478325e1219a6de3308c321abb64862dbc0f128f
Patch: 1fbd486ad934d488b4bd7b6b9085f8f6ebc9b04c91c01c69cf5ced0450862e4b
--- lib/%project_name%/%plugin_name%_plugin.rb.tt	2012-09-04 20:15:42.005614403 +1000
+++ lib/%project_name%/%plugin_name%_plugin.rb.tt	2012-09-04 20:15:45.662303508 +1000
@@ -28,14 +28,6 @@
     feature_manifest ({
     })
 
-    # Callback that is run after the plugin is installed.
-    #
-    # Generator actions are available and can be invoked as if the callback function were a generator method.
-    after_installed {
-      add_dependency_to_gemfile '<%= project_name %>'
-      run_bundle
-    }
-
     # A list of features to install automatically when the plugin itself is installed.
     #
     # @return [Array<String>] An array of feature names. Do not include the plugin prefix.
